dir.props

```diff
@@ -267,6 +267,7 @@
     <IntermediateOutputRootPath Condition="'$(IntermediateOutputRootPath)' == ''">$(BaseIntermediateOutputPath)$(OSPlatformConfig)\</IntermediateOutputRootPath>
     <IntermediateOutputPath Condition="'$(IntermediateOutputPath)' == ''">$(IntermediateOutputRootPath)$(MSBuildProjectName)\</IntermediateOutputPath>
 
+    <ForPackagingDir>$(IntermediateOutputRootPath)sign/forPackaging/</ForPackagingDir>
```

Comment:
How do you feel about just updated the existing `IntermediateOutputForPackaging` property?

[https://github.com/dotnet/core-setup/blob/8bda9c4328e1d59d884799b2375d8c04d8fcb36f/src/dir.props#L7](https://github.com/dotnet/core-setup/blob/8bda9c4328e1d59d884799b2375d8c04d8fcb36f/src/dir.props#L7)

It doesn't appear to be used anywhere.  At least then we don't have 2 "forPackaging" folders.